docs/moments.rst

```diff
@@ -71,5 +83,15 @@ astronomy), you can use::
     >>> sigma_map = cube.linewidth_sigma()  # doctest: +SKIP
     >>> fwhm_map = cube.linewidth_fwhm()  # doctest: +SKIP
 
+The ``linewdith_sigma`` computes a sigma linewidth map along the spectral axis, whereas
+the ``linewidth_fwhm`` function computes a FWHM linewidth map along the same spectral axis.
```

Comment:
ditto here

Reply:
@keflavich Sorry, didn't get the email notification(s) regarding the last requests for changes, so was not aware of these... My apologies.